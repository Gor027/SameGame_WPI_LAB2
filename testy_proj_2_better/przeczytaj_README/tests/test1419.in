U  UUS SUUSS U USUUUUSUSSUS    SU
  S  US U USSS U 
 U S  S  USS USSUUUUU
S SS    U
SU  S U UUSSS SUU   U   SUU S  SS SSU   SS USS   U UUS
UUUU US
SU USUU UUUSU USUSU SU   USUUSU   SUUUU SSSU USSUU
 S U
SU  U USU  SUU SU USS UUUUSUS USS  UUS S US  SS S SUS
SUUS S
SS SSS S S US  U UUU    U S UUU UUU USUS SUS UU SSSU
 SUUUSS 
 U U UUS UUS UUS
