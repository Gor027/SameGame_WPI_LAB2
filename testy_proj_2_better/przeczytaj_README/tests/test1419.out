   U
   S
 U S SS    S   U
 U U US    S   U
 U UUUU UUSS   S
 USUSUS UUUSSU US U U     S
USSSSSUSUUSSSSSSUUUUU   SUU S   U   U    S   S
SUUSUSUSUUSSUUSSUUUUU   SUS U USS SSUU   SSU US UU U
SUUUSSSSUSUUUUUUUSUSSSUSUUSUSSUSS SUUSUSSUSU SS SUSUU
UUUUUUUSUUUSSUUSUUUUSUUUUUSUUUUSUUUUUSUSSSUSSUUSSSSUSS
